server/app/models/Applicant.java

```diff
@@ -107,4 +108,14 @@ public Applicant saveAndReturn() {
     save();
     return this;
   }
+
+  /**
+   * Clears applicantData so that a new ApplicantData object will be returned from getApplicantData.
+   * Useful in tests when applicantData has been stored as an empty object and needs to be udpated
+   * with the data stored in object
```

Comment:
**nitpick:** comment style, also please use `{@link ApplicantData}` and `{@code getApplicantData}` where applicable for easier javadoc navigation